<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MPCT Cards</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/img/icon.ico</normaloff>:/icon/img/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/img/bg.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>720</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/img/null.png);
font: 50pt &quot;交通标志专用字体&quot;;
text-align: center;
color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string>M P C T   Cards</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zhishi_go">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>100</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/img/zhishi.png);
border-radius: 15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="duizhan_go">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>100</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/img/duizhan.png);
border-radius: 15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shiyanshi_go">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>100</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/img/shiyanshi_disabled.png);
border-radius: 15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shangcheng_go">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>231</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/img/shangcheng.png);
border-radius: 15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="beibao_go">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>400</y>
      <width>221</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/img/beibao.png);
border-radius: 15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>zhishi_go</zorder>
   <zorder>shiyanshi_go</zorder>
   <zorder>shangcheng_go</zorder>
   <zorder>beibao_go</zorder>
   <zorder>title</zorder>
   <zorder>duizhan_go</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-image: url(:/image/img/null.png);</string>
   </property>
   <widget class="QMenu" name="user_menu">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="login"/>
    <addaction name="separator"/>
    <addaction name="settings"/>
   </widget>
   <widget class="QMenu" name="files_menu">
    <property name="title">
     <string>存档</string>
    </property>
    <addaction name="read_file"/>
    <addaction name="write_file"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="help"/>
    <addaction name="separator"/>
    <addaction name="about_ext"/>
   </widget>
   <addaction name="user_menu"/>
   <addaction name="files_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <action name="login">
   <property name="text">
    <string>登录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="retisger">
   <property name="text">
    <string>注册</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="read_file">
   <property name="text">
    <string>读档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="write_file">
   <property name="text">
    <string>存档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="about_ext">
   <property name="text">
    <string>关于插件</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
